docker-compose.yaml

```diff
@@ -22,7 +22,7 @@ services:
       context: .
       dockerfile: ./docker/Dockerfile
       args:
-        - DISPLAY=${DISPLAY:-}
+              #- DISPLAY=${DISPLAY:-}
```

Comment:
Please delete commented line

Reply:
Uncommented the DISPLAY arg, as it will be used by heart disease UI client